scenarios/aws/ec2/os_resolver.go

```diff
@@ -92,6 +96,11 @@ func resolveAmazonLinuxAMI(e aws.Environment, osInfo *os.Descriptor) (string, er
 		paramName = fmt.Sprintf("amzn2-ami-hvm-%s-gp2", osInfo.Architecture)
 	case os.AmazonLinuxECS2023.Version:
 		paramName = fmt.Sprintf("al2023-ami-kernel-default-%s", osInfo.Architecture)
+	case os.AmazonLinux2018.Version:
+		if osInfo.Architecture == os.ARM64Arch {
+			return "", errors.New("ARM64 is not supported for Amazon Linux 2018")
+		}
+		return "ami-051394ccb12210d6e", nil // Image name: amzn-ami-2018.03.20240131-amazon-ecs-optimized
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Did you copy this ami and add it to the custom AMIs? Referencing a public AMI comes with the risk of AWS removing it suddenly

Reply:
I didn't. How can I make sure the AMI ID is the same between multiple AWS accounts (agent-sandbox / agent-qa)